numpy/doc/glossary.py

```diff
@@ -430,14 +430,17 @@
          'alpha'
 
    ufunc
-       Universal function.  A fast element-wise array operation.  Examples include
-       ``add``, ``sin`` and ``logical_or``.
-
-   vectorized
-       Optimizing a loop-based function that operates on data with fixed
-       strides via specialized hardware. Compilers know how to take advantage
-       of well-constructed loops and match the data to specialized hardware
-       that can operate on a number of operands in parallel.
+       Universal function.  A fast element-wise, :term:`vectorized
+       <vectorization>` array operation.  Examples include ``add``, ``sin`` and
+       ``logical_or``.
+
+   vectorization
+       Optimizing a looping block by specialized code. In a traditional send,
+       vectorization operates on data with fixed strides via specialized
+       hardware. Compilers know how to take advantage of well-constructed loops
+       and match the data to specialized hardware that can operate on a number
+       of operands in parallel. NumPy uses :ref:`vectorization
+       <whatis-vectorization>` to mean any optimization via specialized code. 
```

Comment:
Specifically it has to be an optimization that involves performing the same operation over multiple elements

Reply:
rephrased